core/trino-parser/src/test/java/io/trino/sql/parser/TestSqlParser.java

```diff
@@ -3255,6 +3256,46 @@ public void testRenameMaterializedView()
         assertStatement("ALTER MATERIALIZED VIEW IF EXISTS a RENAME TO b", new RenameMaterializedView(QualifiedName.of("a"), QualifiedName.of("b"), true));
     }
 
+    @Test
+    public void testSetMaterializedViewProperties()
+    {
+        assertStatement(
+                "ALTER MATERIALIZED VIEW a SET PROPERTIES (foo='bar')",
```

Comment:
brackets `(` and `)` should not be needed,
add tests without them (See `io.trino.sql.parser.TestSqlParser#testSetTableProperties`)